fiftyone/core/odm/utils.py

```diff
@@ -325,6 +337,23 @@ def get_field_kwargs(field):
     return kwargs
 
 
+def _parse_field_str(field_str):
+    chunks = field_str.strip().split("(", 1)
```

Comment:
Fancy! Maybe a comment for readability and future eyes 👀 
```suggestion
    # e.g. fiftyone.core.fields.EmbeddedDocumentField(fiftyone.core.labels.Detections)
    chunks = field_str.strip().split("(", 1)
```

Reply:
👍 added